From: Ben Hutchings <ben@decadent.org.uk>
Subject: fs-writeback: Avoid ABI changes in 3.2.32
Forwarded: not-needed

For some reason, adding an EXPORT_SYMBOL in the middle of fs-writeback.c
changes versions for the later exports.  I can't think why this is, and
I'm sure this normally doesn't cause any problems, but I'm not going to
spend time investigating further.

--- a/fs/fs-writeback.c
+++ b/fs/fs-writeback.c
@@ -63,7 +63,6 @@ int writeback_in_progress(struct backing
 {
 	return test_bit(BDI_writeback_running, &bdi->state);
 }
-EXPORT_SYMBOL(writeback_in_progress);
 
 static inline struct backing_dev_info *inode_to_bdi(struct inode *inode)
 {
@@ -1406,3 +1405,6 @@ int sync_inode_metadata(struct inode *in
 	return sync_inode(inode, &wbc);
 }
 EXPORT_SYMBOL(sync_inode_metadata);
+
+/* Placed at the bottom because it changes versions for the other symbols(!) */
+EXPORT_SYMBOL(writeback_in_progress);
